docker/Dockerfile

```diff
@@ -23,6 +24,8 @@ COPY requirements.txt requirements.txt
 COPY mount.sh mount.sh
 COPY ./src/ ./src/
 
+COPY ./data /app
```

Comment:
remote this. Not everyone has their data in ./data, which would lead to errors when using the docker file

Reply:
shouldn't be there